generators/python_functions/html_unescape/code_snippet.md

```diff
@@ -0,0 +1,12 @@
+```python
+import html
+
+YOUR_ATTRIBUTE: str = "text" #only text attributes
+
+def html_cleanser(record):
```

Comment:
Please ensure that the function name is the same in the `code_snippet.md` and the `__init__.py`! So this should also be `html_unescape`.